cmake/platform/toolcfg/gnu.cmake

```diff
@@ -0,0 +1,21 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2017, 2017
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+
+
+set(OMR_WARNING_AS_ERROR_FLAG -Wall)
```

Comment:
that being said. We probably want to add `-Wall` to all the gcc builds. Regardless of whether `OMR_WARNINGS_AS_ERRORS` is set